core/src/main/java/co/aikar/taskchain/TaskChain.java

```diff
@@ -183,7 +177,7 @@ public static void abort() {
     /**
      * Allows you to call a callback to insert tasks into the chain without having to break the fluent interface
      *
-     * Example: Plugin.newChain().sync(some::task).configure(chain -> {
+     * Example: Plugin.newChain().sync(some::task).configure(chain -&gt; {
```

Comment:
What happened to "->" here?

Reply:
Javadocs are in HTML, so > is a restricted character. &gt; is the replacement for > in HTML.